packages/core/src/SnackbarProvider/SnackbarProvider.styles.ts

```diff
@@ -15,7 +16,7 @@ export const { useClasses, staticClasses } = createClasses(
         fontFamily: "inherit",
         fontWeight: "inherit",
         lineHeight: "inherit",
-        borderRadius: "0",
+        borderRadius: theme.radii.none,
```

Comment:
Can we leave these as they were? We gain nothing from this...

Reasons:
- we're adding a new external dependency on the `theme` object
- the output will be `border-radius: var(--uikit-radii-none)` instead of simply `border-radius: 0`
- consistency. we're not using `theme.radii.none` in other places (apart from a single instance in `BannerContent.styles.tsx`)

The "none" is indeed useful for using with utility classes and components props:
```tsx
<div className="rounded-none" />
<HvButton radius="none" />
```